script/Ciel/Day 8/08_01_CIEL08_6.txt

```diff
@@ -619,7 +619,7 @@ Even though 'I' become the owner of the body,
 -- Page 534, Offset 19631.
 -- つまり―――わたしの肉体は『私』という転生者のものだったけれど、意識はもとのわたしのままだった。
 // Back to Ciel
-Which is to say: though my body may have belonged to the reincarnated 'me', my consciousness was still mine.
+Which is to say, though my body may have belonged to the reincarnated 'me', my consciousness was still mine.
```

Comment:
Prefer colon

Reply:
colon can be replaced with dash at worst
otherwise keep it